test/e2e/query_frontend_test.go

```diff
@@ -5,22 +5,29 @@ package e2e_test
 
 import (
 	"context"
+	"crypto/sha256"
 	"fmt"
+	"net/http"
+	"net/http/httptest"
 	"os"
 	"reflect"
 	"sort"
 	"testing"
 	"time"
 
+	"github.com/prometheus/prometheus/model/labels"
+	"github.com/thanos-io/thanos/pkg/tenancy"
```

Comment:
nit: can probably sort these in the pre-existing pattern?

Reply:
These are already formatted (and thus sorted) by `go fmt` and `goimports`, otherwise linting would've failed.

Reply:
Redid the grouping and formatted imports with `goimports` again.